### PR TITLE
add binstub and allow shebang

### LINKED ISSUE #675: Include binstub for *nix in build artifacts
via @paulcbetts https://gist.github.com/paulcbetts/e8514a03c6c01d1677cb

```bash
#!/usr/bin/env mono
## Make sure to chmod +x this file!

DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"

mono "$DIR/scriptcs.exe" $@
```

### PR DESCRIPTION
**Pull request by adamralph**
fixes #675 and #956

Raised on behalf of @mbaker-dev, even if just for discussion/review.

### CODE REVIEW

File: scriptcs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+## Make sure to chmod +x this file!
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+mono "$DIR/scriptcs.exe" $@
```
Comment (adamralph) [inline scriptcs]: I think this should probably live under `/src/ScriptCs` with `Copy to Output Directory` set to `Copy Always`, so it just gets emitted into `bin/Release` along with everything else.

**Comment by adamralph** [comment]
I think this looks great! Perhaps we should just tidy up the location of the binstub and then I'd say it's good to merge!

**Comment by mbaker-dev** [comment]
I've moved the binstub to src/ScriptCs and included it in the csproj with CopyToOutputDirectory=Always. It's including the file in the build output now, as well as the artifacts area on Windows and *nix; however, it's not being included in the NuGet package. If it's included in the NuGet package, I think it should be specifically excluded from the Chocolatey install?

**Comment by adamralph** [comment]
@mbaker-dev thanks again. Just having the binstub in the artifacts is a step forward. At the moment we don't provide any packages for Linux or Mac, but I've made a note on #716 and #972 that we now have the binstub in the artifacts.

**Comment by adamralph** [comment]
I've added #675 and #956 to the v0.14 release notes.

**Comment by BrendanThompson** [comment]
This is getting exciting!

**Pull request merged**